src/main/java/com/iota/iri/service/dto/GetNeighborsResponse.java

```diff
@@ -25,7 +23,7 @@
      *     <li>numberOfSentTransactions</li>
      *     <li>numberOfStaleTransactions</li>
      * </ol>
-     * @see {@link com.iota.iri.service.dto.GetNeighborsResponse.Neighbor}
+     * @see com.iota.iri.service.dto.GetNeighborsResponse.Neighbor
```

Comment:
@kwek20 
Can you verify that your doclet works properly with those changes?

Reply:
Confirmed this is OK! Its still seen as a class link because every `@see` is a link